src/modules/partition/core/DirFSRestrictLayout.cpp

```diff
@@ -0,0 +1,234 @@
+/* === This file is part of Calamares - <https://calamares.io> ===
+ *
+ *   SPDX-FileCopyrightText: 2014-2017 Teo Mrnjavac <teo@kde.org>
+ *   SPDX-FileCopyrightText: 2017-2018 Adriaan de Groot <groot@kde.org>
+ *   SPDX-FileCopyrightText: 2018-2019 Collabora Ltd <arnaud.ferraris@collabora.com>
```

Comment:
This is all your own work, so please put in a single line for yourself, and leave these out since they simply are not true.

Reply:
It's not strictly my work, I copied an existing file and then modified it until it did what I wanted, so I think the existing attributions are accurate. I'm happy to add my own too though.

Reply:
Added.